apps/pwabuilder/src/script/pages/app-report.ts

```diff
@@ -2439,11 +2441,7 @@ export class AppReport extends LitElement {
     this.showConfirmationModal = false;
 
     details.forEach((detail: any) => {
-      if(detail.id != "todo-detail"){
```

Comment:
I prefer https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality, but in this case its ok